vlib/v/gen/native/tests/print.vv

```diff
@@ -110,6 +110,18 @@ fn test_intermediate() {
 	println(c)
 }
 
+fn test_for_range() {
+	for i in 0 .. 10 {
+		println(i)
+	}
```

Comment:
add also a separate loop for `for i in 7 .. 9 {`
and also a separate loop for `for i in -2 .. 3 {`
to check for more common cases